jest.config.ts

```diff
@@ -4,7 +4,41 @@
  */
 
 import { Config } from '@jest/types'
+
 import makeConfig from '@hashicorp/platform-configs/jest/config.js'
+import { compilerOptions } from './tsconfig.json'
+
+const tsPathsToModuleNameMapper = (mapping, baseUrl) => {
```

Comment:
We are not using pathsToModuleNameMapper from ts-jest because:
1. It does not support unix backwards paths `../`